packages/api-client/src/Document/Document.spec.ts

```
@@ -165,9 +164,12 @@ describe('Document', () => {
         original_hash: 'some-file',
       });
 
-      expect(requestMock.mock.calls[0][0].headers['content-type']).toContain(
-        'multipart/form-data'
-      );
+      expect(requestMock.mock.calls[0][0].data).toBeInstanceOf(FormData);
+      const { headers } = requestMock.mock.calls[0][0];
+      const contentType =
+        headers['content-type'] ?? headers['Content-Type'] ?? headers.get?.('Content-Type');
```

ESLint

[error] Replace `·headers['Content-Type']·??` with `⏎········headers['Content-Type']·??⏎·······`

(prettier/prettier)